charts/rsyncd/values.yaml

```
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 image:
   repository: jenkinsciinfra/rsyncd
-  tag: 1.0.67
+  tag: 2.0.0
```

```suggestion
  tag: 1.0.68
```

=> last "non breaking change" version, with only a debian base image bump